app/controllers/admin/conferences_controller.rb

```diff
@@ -72,11 +72,15 @@ def index
 
     def new
       @conference = Conference.new
+      @organizations = {}
+      Organization.find_each do |organization|
```

Comment:
yeah that looks much better!
I'm just wondering, why do you use `Hash#store` instead of `Hash#[]=`?
It's not common practice, so it looks kinda funky :P